cmc-ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/util/AssertUtil.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.ccd.mapper.util;
+
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails;
+import uk.gov.hmcts.cmc.domain.models.legalrep.Representative;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AssertUtil {
+
+    private AssertUtil() {
+    }
+
+    public static void assertAddressEqualTo(Address address, uk.gov.hmcts.cmc.ccd.domain.Address ccdAddress) {
```

Comment:
Either implement a [custom AssertJ assertion](http://joel-costigliola.github.io/assertj/assertj-core-custom-assertions.html#custom-assertion-class) or an `equals` method in the model and use `isEqualTo` assertion. Applies to all methods of this class and this class should be removed.